server/src/Application/HumanResource/Leave/Command/UpdateLeaveRequestCommandHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { Inject } from '@nestjs/common';
+import { CommandHandler } from '@nestjs/cqrs';
+import { ILeaveRequestRepository } from 'src/Domain/HumanResource/Leave/Repository/ILeaveRequestRepository';
+import { UpdateLeaveRequestCommand } from './UpdateLeaveRequestCommand';
+import { LeaveRequestNotFoundException } from 'src/Domain/HumanResource/Leave/Exception/LeaveRequestNotFoundException';
+
+@CommandHandler(UpdateLeaveRequestCommand)
+export class UpdateLeaveRequestCommandHandler {
+  constructor(
+    @Inject('ILeaveRequestRepository')
+    private readonly leaveRequestRepository: ILeaveRequestRepository
+  ) { }
+
+  public async execute(command: UpdateLeaveRequestCommand): Promise<string> {
+    const {
+      endDate,
+      endsAllDay,
+      id,
+      type,
+      startDate,
+      startsAllDay,
+      comment
```

Comment:
(Nit) Respecter l'ordre des champs

```suggestion
      id,
      type,
      startDate,
      startsAllDay,
      endDate,
      endsAllDay,
      comment
```